docs/usage.rst

```diff
@@ -2,44 +2,27 @@
 Running a simulation
 ====================
 
-``pyuvsim`` takes a UVData object, instrument configuration settings, and a source catalog and effectively "fills" the UVData object with simulated data. The function ``run_pyuvsim`` in uvsim.py
+``pyuvsim`` takes a UVData object, instrument configuration settings, and a source catalog and effectively "fills" the UVData object with simulated data. The function ``uvsim.run_pyuvsim`` in uvsim.py accepts as input a path to a yaml ``obsparam`` file and writes out the data to ``uvfits`` format. Optionally, it can also skip writing the data out and just return a ``UVData`` object filled with simulated data.
```

Comment:
This text still suggests to me that a UVData object is a standard input. Is that what we want?

Reply:
I wanted to emphasize that the underlying function of pyuvsim is to "fill" a UVData object, but I can rephrase it to avoid giving the impression that uvdata files are supported as input.